coalib/output/ConsoleInteraction.py

```diff
@@ -289,7 +289,8 @@ def print_results_formatted(log_printer,
     default_format = ('id:{id}:origin:{origin}:file:{file}:line:{line}:'
                       'column:{column}:end_line:{end_line}:end_column:'
                       '{end_column}:severity:{severity}:severity_str:'
-                      '{severity_str}:message:{message}')
+                      '{severity_str}:message:{message}:affected_code:'
+                      '{affected_code}')
```

Comment:
you change the default format, this would
1. belong to a different commit
2. need discussion/issue or at least a good reason why you change it